frame/mosaic/src/lib.rs

```diff
@@ -353,9 +353,14 @@ pub mod pallet {
 
 	#[pallet::call]
 	impl<T: Config> Pallet<T> {
-		/// Sets the current relayer configuration. This is enacted immediately and invalidates
-		/// inflight, incoming transactions from the previous relayer. Budgets remain in place
-		/// however.
+		/// Sets the current Relayer configuration.
+		///
+		/// This is enacted immediately and invalidates inflight/ incoming transactions from the
+		/// previous Relayer. However, existing budgets remain in place.
+		///
+		/// This can only be called by the [`ControlOrigin`].
+		///
+		/// [controlorigin]: https://dali.devnets.composablefinance.ninja/doc/pallet_mosaic/pallet/trait.Config.html#associatedtype.ControlOrigin
```

Comment:
You can have relative links in cargo docs
See my example [here](https://github.com/ComposableFi/composable/blob/fd6649d63883cab4daf3d1ae238b749ba01232b0/frame/mosaic/src/lib.rs#L359)
And a more general note of the feature from the rust team [here](https://rust-lang.github.io/rfcs/1946-intra-rustdoc-links.html)

Reply:
Though, I suppose this is better as if for when we aren't in cargo docs